src/Selenium2Library/keywords/_element.py

```diff
@@ -296,7 +296,21 @@ def get_horizontal_position(self, locator):
         if element is None:
             raise AssertionError("Could not determine position for '%s'" % (locator))
         return element.location['x']
-
+ 
+    def get_element_size(self, locator):
+        """Returns width and height of element identified by `locator`.
+            
+        The height abd height and width is returned in pixels,as an integer.
+        Fails if a matching element is not found.
+        
+        """
+        element = self._element_find(locator, True, False)
```

Comment:
Perhaps the required argument should be set to True. Then ```_element_find``` will raise ValueError with better error message. Also then lines 308 - 309 are not needed